mscore/scoretab.cpp

```diff
@@ -555,26 +557,26 @@ void ScoreTab::removeTab(int idx, bool noCurrentChangedSignal)
 //   initScoreView
 //---------------------------------------------------------
 
-void ScoreTab::initScoreView(const int idx, const qreal logicalZoomLevel, const ZoomIndex zoomIndex, const qreal xoffset, const qreal yoffset)
+void ScoreTab::initScoreView(int idx)
       {
-      ScoreView* v = view(idx);
-      if (!v)  {
-            v = new ScoreView;
-            Score* sc = scoreList->value(idx);
-            if( sc != 0 )
-                  v->setScore(sc);
```

Comment:
two notes here:
- I see like previously we were creating a new ScoreView if we couldn't get it from view(idx)
- Also I can't find a place where we're setting a Score for ScoreView

I assume it's happening somewhere else, but it's not really obvious now how it works

Reply:
I believe it works like this:
- The tab is added in `restoreSession` / `openScore` / `appendScore` (but the score view not yet initialized)
- When the user switches to a tab, via signals/slots `ScoreTab::setCurrent` gets called, where the ScoreView is initialized and the score is set. 
- This function, `ScoreTab::initScoreView` is called at the bottom of `ScoreTab::setCurrent`, where we have guarantee that the scoreView is initialized.